runners/flink/flink_runner.gradle

```diff
@@ -34,6 +34,11 @@ applyJavaNature(
 
 description = "Apache Beam :: Runners :: Flink $flink_version"
 
+dependencies {
+  implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-jaxb-annotations', version: '2.12.3'
```

Comment:
Hello, thanks for contributing this fix! Which version of Flink (EMR) were you able to run with this fix (just out of curiosity).

Can you put this on the main dependencies block
https://github.com/apache/beam/blob/92386d781b5d502c4ea47a0894b72ca57854553d/runners/flink/flink_runner.gradle#L176

And can you use the default library definitions (and add the jsr310 one there).
https://github.com/apache/beam/blob/92386d781b5d502c4ea47a0894b72ca57854553d/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L580

Reply:
I tested this on EMR 6.3.0 and 5.33